display/d.labels/do_labels.c

```diff
@@ -452,7 +465,11 @@ int scan_ref(char *buf)
         if (buf[i] >= 'A' && buf[i] <= 'Z')
             buf[i] += 'a' - 'A';
     xref = yref = CENT;
-    switch (sscanf(buf, "%s%s", word1, word2)) {
+    char word_fmt[10];
+    snprintf(word_fmt, sizeof(word_fmt), "%%%ds%%%ds", WORDSIZE - 1,
+             WORDSIZE - 1);
```

Comment:
No need for this if word_fmt is declared in file scope.

Reply:
done